### ISSUE TITLE
feat: apply finetuned sequence model to get `pred_proba_series`

### ISSUE THREAD

**Issue by MartinBernstorff** [Aarhus-Psychiatry-Research/psycop-common#535]
Perhaps from a finetuned checkpoint?

**Comment by MartinBernstorff** [comment]
Thinking about this for a bit, perhaps something like:

GIVEN a finetuned checkpoint
AND a cohort definer
AND a way to split the cohort definer
WHEN we call the function/method
THEN we get a PolarsFrame containing `patient_id` and `pred_proba`

That'd be my desired behaviour from the outside, but we most likely want to decompose it somewhat.

**Comment by MartinBernstorff** [comment]
Looking at our `PatientSliceClassifier`, I assume we basically need the `torch.sigmoid(logits)` as output.

I wonder whether this is best accomplished by:
* Refactoring the forward pass, so that getting the logits is its own method
* Adding a `_calculate_probabilities` method which
    * Calculates logits
    * Converts them to probs
    * Returns the probs
* Refactor the `_calculate_metrics` metrics function to get the probs from the `calculate_probabilities` method

**Comment by MartinBernstorff** [comment]
Probably not required for the MVP, since we get AUROC from pytorch metrics. Closing, feel free to reopen if you disagree @KennethEnevoldsen.

**Comment by KennethEnevoldsen** [comment]
Isn't this something we have already implemented. Should be easy enough to run.

**Comment by MartinBernstorff** [comment]
I don't think we've written code for this. You were assigned to it, but this issue isn't referenced anywhere. If you can point to some code, I'll gladly test it 👍
